### ISSUE TITLE
Upgrade LND 

### ISSUE THREAD

**Issue by BTCxZelko** [raspiblitz/raspiblitz#824]
With Upgrade 0.8.0 out. How can we go about upgrading?

**Comment by lutzk87** [comment]
Upgrade guide from openoms. worked for me!

> https://twitter.com/openoms/status/1184201843574034433?s=20

https://github.com/openoms/lightning-node-management/blob/master/lnd.updates/README.md

**Comment by PBJellytime** [comment]
Upgrade worked for me, however when I try and view the channels now I get the below error message 

[lncli] credentials: failed to append certificates
[lncli] credentials: failed to append certificates
[lncli] credentials: failed to append certificates
[lncli] credentials: failed to append certificates
[lncli] credentials: failed to append certificates
[lncli] credentials: failed to append certificates
[lncli] credentials: failed to append certificates

mainnet                            Commit ------- Balance --------- --- Fee ----
Alias or Pubkey           Capacity   Fee        Local       Remote   Base PerMil
-------------------- ------------- ------ ------------ ------------ ----- ------
-------------------- ------------- ------ ------------ ------------ ----- ------
[lncli] credentials: failed to append certificates
Totals          0 ch             0      0            0            0 Day:
[lncli] credentials: failed to append certificates
                                                                    Week:
[lncli] credentials: failed to append certificates
                                                                    Month:

**Comment by openoms** [comment]
@PBJellytime 
not sure what this error is, can you give more context?
Is it only the `lnchannels` command failing?
Is your node functional otherwise?

**Comment by rootzoll** [comment]
@openoms can you confirm that the lnchannels is still working after the upgrade to lnd 0.8?

**Comment by openoms** [comment]
> @openoms can you confirm that the lnchannels is still working after the upgrade to lnd 0.8?

Yes, `lnchannels` work perfectly on all my updated nodes.

**Comment by openoms** [comment]
The new Bitcoin Core v0.19.01 is out https://twitter.com/bitcoincoreorg/status/1198559459401965568

LND v0.8.1 is an update to make it compatible with the above so we can update both for the next release.

**Comment by frennkie** [comment]
Appears to working fine. Just on "gotcha": LND v0.8.1-beta identifies as v0.8.0-beta

```
$ lnd --version
lnd version 0.8.0-beta commit=v0.8.1-beta
$ sha256sum /usr/local/bin/lnd
654f222432dc3bae806032a4939d97eefc29e05d2ff380a5b9586ba6897dbc85  /usr/local/bin/lnd
```
They forgot to increase the version counter.

I updated the sd_build.sh script with: ff18754

**Comment by openoms** [comment]
@frennkie did you do a manual SDcard build to test?

**Comment by frennkie** [comment]
No, but will do so now.

**Comment by openoms** [comment]
The update script works as well with LND v0.8.1-beta but the status screen continues to display `LND v0.8.0-beta`
https://github.com/openoms/raspiblitz-extras#lnd-update-to-v081-beta

**Comment by openoms** [comment]
The latest LND is in the master branch now. Update scripts functional. Closing.

**Comment by rootzoll** [comment]
LND 0.8.2 is already in RC2 ... expected to be final for v1.4 release. So I reopen this issue to check again closer to v1.4 release: https://github.com/LightningNetwork/lnd/releases

**Comment by openoms** [comment]
Running LND v0.8.2-beta on two nodes. There was no issue with the update. https://twitter.com/openoms/status/1205250222164713472

**Comment by rootzoll** [comment]
Tests look good with 0.8.2 ... closing issue.

**Comment by openoms** [comment]
Upgrade to 0.9.0-beta-rc1 went ok; https://twitter.com/openoms/status/1216428426724356098

Update script: https://github.com/openoms/lightning-node-management/tree/master/lnd.updates#update-lnd-to-v090-beta-rc1

**Comment by rootzoll** [comment]
Yeah looks like we need some extra days anyway to get rid of the bugs of v1.4 ... so why not wait for the final LND v.0.9 release.

**Comment by openoms** [comment]
Running the LND 0.9.0-beta-rc3 looking good.

**Comment by rootzoll** [comment]
Final LND 0.9 release seems right around the corner.

**Comment by rootzoll** [comment]
LND 0.9.0 is out .. adding it to the build script :D
https://github.com/lightningnetwork/lnd/releases/tag/v0.9.0-beta

**Comment by openoms** [comment]
> LND 0.9.0 is out .. adding it to the build script :D
> https://github.com/lightningnetwork/lnd/releases/tag/v0.9.0-beta

And running it already, updated smoothly: https://github.com/openoms/lightning-node-management/tree/master/lnd.updates#update-lnd-to-v090-beta

**Comment by openoms** [comment]
@rootzoll watch the signing keys it is @roasbeef who signed the release despite the Release notes pointing to Joost`s PGP: https://twitter.com/openoms/status/1220041314743595008

**Comment by rootzoll** [comment]
OK - tests also run good on my machine. v1.4 will be released with LND v0.9.0-beta
closing issue
